src/main/resources/application.yml

```diff
@@ -1,13 +1,14 @@
 spring:
   profiles:
-    active: 'prod'
+    active: 'dev'
```

Comment:
Should not be commited with active dev profile

Reply:
I've done changes and pushed new file application.yml